resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java

```diff
@@ -0,0 +1,56 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+import com.google.sps.data.Reviewee;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that saves reviewer data from the form */
+@WebServlet("/reviewee-data")
+public class RevieweeDataServlet extends HttpServlet {
+
+  private Reviewee reviewee;
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Send the JSON as the response
+    response.setContentType("application/json");
+    String json = new Gson().toJson(reviewee);
+    response.getWriter().println(json);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String fname = getParameter(request, "fname", "");
+    String lname = getParameter(request, "lname", "");
+    String email = getParameter(request, "email", "");
+    reviewee = new Reviewee(fname, lname, email);
+    Entity revieweeEntity = new Entity("Reviewee");
+    revieweeEntity.setProperty("first-name", fname);
+    revieweeEntity.setProperty("last-name", lname);
+    revieweeEntity.setProperty("email", email);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(revieweeEntity);
+
+    /* TODO: add a response.sendRedirect() to wherever the user should
+     * be navigated to next (either homepage or resume review page)
+     */ 
+  }
+
+  /**
+   * @return the request parameter, or the default value if the parameter was not specified by the
+   * client
+   */
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+    String value = request.getParameter(name);
+    return (value == null) ? defaultValue : value;
+  }
```

Comment:
Nit: You'll probably use this helper function in all HttpServlet classes in the project. If that's the case, you'll duplicate this function in every single HttpServlet.

What you can do is create a utility class dedicated to static helper functions. The class could be named `ServletHelpers` or something like that. This function should be a public static function on that class. The class should have a private constructor, which means one can't create instances of it. 

This is totally optional, btw. It's not a huge deal, mostly because there's only this helper function right now. Over time you may write more helper functions that can serve multiple servlet classes and can be static. Then you'd definitely want a utility class so your servlet classes don't have so much duplicate code between them.

Reply:
Okay, I'll do this in a new PR!